app/helpers/utils.js

```diff
@@ -11,8 +11,14 @@ const isValidPassword = (password, passwordEncrypted) => bcryptjs.compareSync(pa
 
 const generateToken = payload => jwt.encode(payload, process.env.JWT_KEY_SECRET);
 
+const decodeToken = token => jwt.decode(token, process.env.JWT_KEY_SECRET);
+
+const compareData = (param1, param2) => JSON.stringify(param1) === JSON.stringify(param2);
```

Comment:
dont use numbers at variables use something different.

remember to use something in the function that its name indicates its return type value. Eg. a boolean can be with `is` because the answer can be `yes|no` 

```suggestion
const isObjectEqual = (param1, param2) => JSON.stringify(param1) === JSON.stringify(param2);
```